### PR TITLE
Fix external CSS stylesheet links for html files within doc sub-directories

### PR DESCRIPTION
**Pull request by Anirban166**
The links to the estilo1 CSS file mentioned in the html files within the folders inside doc/ (just saw this while browsing through them) were also linked to the rforge hosted file (http://directlabels.r-forge.r-project.org/estilo1.css), which is not accessible outside, so I changed them to link back to the CSS file in docs/ (I could have also used the one in /root, should I just keep one copy of the css file?) and did the same in templates/head.html for automatic updation when using dldoc

(For example, you can check https://tdhock.github.io/directlabels/docs/contourplot/posfuns/bottom.pieces.html and http://directlabels.r-forge.r-project.org/docs/contourplot/posfuns/bottom.pieces.html to see the difference before merging this)

@tdhock  If you find anything falling behind or lacking in https://tdhock.github.io/directlabels/ with respect to the rforge hosted website (http://directlabels.r-forge.r-project.org/), then let me know!

### CODE REVIEW

**Comment by tdhock** [comment]
no need for two copies css file. did you find the button / arrow to mark pull request as a draft? https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request

**Comment by Anirban166** [comment]
> no need for two copies css file. did you find the button / arrow to mark pull request as a draft? https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request

Yup thought so, I made the changes and yes, I did find the button, I'll use it for PRs which I'm working on and which are not ready to be merged right away!

**Comment by Anirban166** [comment]
Some more changes to the templates could include:
- Replacing `install.packages(c("directlabels","ggplot2","ElemStatLearn","mlmRev")) ` with `install.packages(c("ggplot2", "directlabels", "reshape2", "mlmRev", "lars", "latticeExtra"))` (excluding inlinedocs, i.e. considering the user would directly want to run dldoc without re-building the skeleton, and ElemStatLearn is no longer required now so removed that) in [templates/index.html](https://github.com/tdhock/directlabels/blob/gh-pages/docs/templates/index.html)
- Replacing `http://inlinedocs.r-forge.r-project.org` with `https://github.com/tdhock/inlinedocs`, since you mentioned inlinedocs from rforge is deprecated (although the landing webpage on rforge is helpful, maybe you should host that via GitHub pages and a link to that would be better?) and maybe replacing the link to your contact in [templates/foot.html](https://github.com/tdhock/directlabels/blob/gh-pages/docs/templates/foot.html)

@tdhock what do you think?

**Comment by tdhock** [comment]
yes please update the templates

**Comment by Anirban166** [comment]
> yes please update the templates

okay, which link of yours should I put for the contact?

**Comment by tdhock** [comment]
http://tdhock.github.io/

**Comment by Anirban166** [comment]
@tdhock are the changes okay? anything else that you would like me to change here on this branch or under this PR? (would need to get this merged before #38 - since that will bring changes into gh-pages)

File: docs/contourplot/posfuns/bottom.points.html [hunk 1]
```diff
@@ -65,13 +65,16 @@ <h1><a href="../../index.html">directlabels</a> - contourplot -
...
+Dylan Hocking</a> if you are
+using <a href="https://tdhock.github.io/directlabels/">directlabels</a>
+or have ideas to contribute, thanks!</td>
 </tr>
-<tr><td align="center">Documentation website generated from comments
-in R source code
-using <a href="http://inlinedocs.r-forge.r-project.org">inlinedocs</a>.</td>
+<tr><td align="center">Documentation website generated from source
+code version 2014.1.27
+(svn revision 675)
```
Comment (tdhock) [inline docs/contourplot/posfuns/bottom.points.html:76]: this should be changed eventually, is now a good time?

**Comment by tdhock** [comment]
it is difficult/impossible for me to review this PR because there are so many changes (103 files changed).
should I just merge without reviewing? (tell me if you are ready to merge)
otherwise if you want my feedback on something in particular can you please comment on the commit/diff line(s) you want me to review?

File: docs/contourplot/posfuns/bottom.points.html (hunk 1 above)
Comment (Anirban166) [reply docs/contourplot/posfuns/bottom.points.html:76]: Yep I'll do it now, thanks for pointing that out!

Comment (Anirban166) [reply docs/contourplot/posfuns/bottom.points.html:76]: Do we need to include the git version info? The current code (lines [75](https://github.com/tdhock/directlabels/blob/8f8a4f27c9eb0d207dd3a01a0d58fae31496753b/R/doc.R#L75) to [77](https://github.com/tdhock/directlabels/blob/8f8a4f27c9eb0d207dd3a01a0d58fae31496753b/R/doc.R#L77)) provides the following information:
image

**Comment by Anirban166** [comment]
> it is difficult/impossible for me to review this PR because there are so many changes (103 files changed).
> should I just merge without reviewing? (tell me if you are ready to merge)
> otherwise if you want my feedback on something in particular can you please comment on the commit/diff line(s) you want me to review?

That's true. The changes are same for each and every html file under plots/ and posfuns/ under different plot-type directories, so you can just check any one file (rest assured, the rest are same). It is ready to be merged!

**Comment by Anirban166** [comment]
Here is the breakdown of changes just to view: (apart from deletion of the copy of estilo1.css)
> The changes are same for each and every html file under plots/ and posfuns/ under different plot-type directories

Changes to such an html file:
image
image

Changes to docs/index.html: 
image
image
image

Changes to files under docs/templates/:
image
image
image

**Comment by tdhock** [comment]
thanks

**Pull request merged**